src/main/webapp/app/localvc/repository-view/repository-view.component.ts

```
@@ -91,11 +91,14 @@ export class RepositoryViewComponent implements OnInit, OnDestroy {
             this.participationCouldNotBeFetched = false;
             const exerciseId = Number(params['exerciseId']);
             const participationId = Number(params['participationId']);
+            const repositoryId = Number(params['repositoryId']);
             this.repositoryType = participationId ? 'USER' : params['repositoryType'];
             this.vcsAccessLogRoute = this.router.url + '/vcs-access-log';
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)